Library/SharedFunctions.swift

```diff
@@ -687,19 +687,15 @@ public func estimatedShippingText(
 
   guard estimatedMin > 0, estimatedMax > 0 else { return nil }
 
-  let currentCountry = project.stats.currentCountry ?? Project.Country.us
```

Comment:
Nit: Since `project.country` is used in multiple places, keep this variable and just assign it to `project.country` instead.